webapp/pages/login.vue

```diff
@@ -7,19 +7,26 @@
 <script>
 import LoginForm from '~/components/LoginForm/LoginForm.vue'
 import { VERSION } from '~/constants/terms-and-conditions-version.js'
+import { mapGetters } from 'vuex'
 
 export default {
   layout: 'no-header',
   components: {
     LoginForm,
   },
+  computed: {
+    ...mapGetters({
+      user: 'auth/user',
+    }),
+  },
   asyncData({ store, redirect }) {
     if (store.getters['auth/user'].termsAndConditionsAgreedVersion === VERSION) {
       redirect('/')
     }
   },
   methods: {
     handleSuccess() {
+      this.$i18n.set(this.user.locale || 'en')
       this.$router.replace(this.$route.query.path || '/')
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-11-04T19:53:14Z" title="Monday, November 4th 2019, 8:53:14 pm +01:00">Nov 4, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -7,19 +7,26 @@
 <script>
 import LoginForm from '~/components/LoginForm/LoginForm.vue'
 import { VERSION } from '~/constants/terms-and-conditions-version.js'
+import { mapGetters } from 'vuex'
 
 export default {
   layout: 'no-header',
   components: {
     LoginForm,
   },
+  computed: {
+    ...mapGetters({
+      user: 'auth/user',
+    }),
+  },
   asyncData({ store, redirect }) {
     if (store.getters['auth/user'].termsAndConditionsAgreedVersion === VERSION) {
       redirect('/')
     }
   },
   methods: {
-    handleSuccess() {
+    handleSuccess() {      
+      this.$i18n.set(this.user.locale)   
```


What about a default here?
```suggestion
      this.$i18n.set(this.user.locale || 'en')   
```

Reply:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-11-06T11:25:22Z" title="Wednesday, November 6th 2019, 12:25:22 pm +01:00">Nov 6, 2019</time>_

---


@ogerly can you please add this?